.github/actions/build/action.yml

```diff
@@ -22,11 +22,13 @@ runs:
     - name: Install dependencies
       shell: bash
       run: pnpm install
-    - name: Turborepo cache
+    - name: Cache turbo build setup
       uses: actions/cache@3624ceb22c1c5a301c8db4169662070a689d9ea8
       with:
         path: .turbo
-        key: turbo-${{ runner.os }}-node-${{ inputs.node-version }}
+        key: ${{ runner.os }}-turbo-${{ github.sha }}
```

Comment:
This is adding the commit hash to the cache key. So new commits can’t benefit from the cache anymore, right?